db/structure.sql

```diff
@@ -2227,6 +2242,9 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20200908173642'),
 ('20200911143301'),
 ('20200922131539'),
-('20200929205630');
+('20200929205630'),
+('20201029160139'),
+('20201102155621'),
+('20201102163702');
```

Comment:
I think you added more migrations here than what you should have. I think we can keep just the last one, `20201102163702`